lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class VersionNumberCubit extends Cubit<String> {
+  VersionNumberCubit() : super('');
+
+  void getVersion() async {
+    final info = await PackageInfo.fromPlatform();
+    emit(info.version);
+  }
+}
```

Comment:
Creating a new cubit is not necessary as we don't have any dynamic properties in the state. Also, we can't update the version number dynamically in Flutter. So It's fine to call `PackageInfo.fromPlatform();` directly in the `VersionNumber` widget.

Reply:
@SankethBK I did not see Future Builder being used anywhere... that's why I created a cubit. Can I use a Future builder? or should I use something else?!!

Reply:
Hey again... I went ahead and used Future builder.

Reply:
Hey `FutureBuilder` is fine